.github/renovate.json5

```diff
@@ -250,6 +258,46 @@
         "docker.io/library/golang"
       ],
     },
+    {
+      "groupName": "all external docker images dependencies",
+      "groupSlug": "all-external-images-deps-main",
+      "matchFiles": [
+        "install/kubernetes/Makefile.values"
+      ],
+      "matchUpdateTypes": [
+        "major",
+        "minor",
+        "digest",
+        "patch",
+        "pin",
+        "pinDigest"
+      ],
+      "schedule": [
+        "on monday"
+      ],
```

Comment:
should this also run on Sundays instead?

Reply:
for cilium/proxy, I have it set as Monday.

Reply:
This was removed https://github.com/cilium/cilium/pull/33531/commits/742d914437ad60c0c8c8286b52616405339b416a